apps/web/components/Proposals.tsx

```diff
@@ -447,20 +478,21 @@ export function Proposals({
     (x) => stakedFilters[x.id]?.value,
   );
 
+  const membersStrategies = membersStrategyData?.memberStrategies;
+
   // Render
   return (
     <>
-      {strategy.isEnabled && (
-        <PoolGovernance
-          memberPoolWeight={memberPoolWeight}
-          tokenDecimals={tokenDecimals}
-          strategy={strategy}
-          communityAddress={communityAddress}
-          memberTokensInCommunity={memberTokensInCommunity}
-          isMemberCommunity={isMemberCommunity}
-          memberActivatedStrategy={memberActivatedStrategy}
-        />
-      )}
+      <PoolGovernance
```

Comment:
why did you removed the stategy.isEnbaled check ? I dont think we want to see that if its pending or archived

Reply:
hmm Im checking ...